Commit 16d17f9 doesn't compile

Commit 16d17f9 doesn't compile on any platform, due to the merge of forks #264 and #265.

```
/home/travis/build/Segs/Segs/Projects/CoX/Servers/MapServer/MapInstance.cpp:791:35: error: 
      use of undeclared identifier 'InfoType'
        info = new InfoMessageCmd(InfoType::DEBUG_INFO, msg);
                                  ^
/home/travis/build/Segs/Segs/Projects/CoX/Servers/MapServer/MapInstance.cpp:792:73: error: 
      reference to non-static member function must be called
        src->addCommandToSendNextUpdate(std::unique_ptr<InfoMessageCmd>(info));
```

#265 added a new `/settingsdebug` command, which needs to be converted to new slashcommand format from #264. Github didn't seem to catch the change.

I'll submit a patch today (Mar 8) to resolve the issue.

--- a/Projects/CoX/Servers/MapServer/MapInstance.cpp
+++ b/Projects/CoX/Servers/MapServer/MapInstance.cpp
@@ -27,7 +27,6 @@
 #include "Database.h"
 #include "SlashCommand.h"
 #include "Common/GameData/CoHMath.h"
-#include "Settings.h"
 
 #include <QtCore/QDebug>
 #include <QtCore/QFile>
@@ -783,13 +782,6 @@ void MapInstance::on_console_command(ConsoleCommand * ev)
             || contents.startsWith("me ",Qt::CaseInsensitive))                                  // ERICEDIT: This encompasses all emotes.
     {
         on_emote_command(lowerContents, ent);
-    }
-    else if(lowerContents == "settingsdump") {
-        settingsDump();
-
-        QString msg = "Sending settings config dump to console output.";
-        info = new InfoMessageCmd(InfoType::DEBUG_INFO, msg);
-        src->addCommandToSendNextUpdate(std::unique_ptr<InfoMessageCmd>(info));
     }
     else {
         runCommand(contents,*ent);

--- a/Projects/CoX/Servers/MapServer/SlashCommand.cpp
+++ b/Projects/CoX/Servers/MapServer/SlashCommand.cpp
@@ -1,6 +1,7 @@
 #include "SlashCommand.h"
 #include "DataHelpers.h"
 #include "MapInstance.h"
+#include "Settings.h"
 
 #include <QtCore/QString>
 #include <QtCore/QFile>
@@ -39,6 +40,7 @@ std::vector<SlashCommand> g_defined_slash_commands = {
     {{"hascontrolid"}, &cmdHandler_HasControlId, 9},
     {{"setTeam"}, &cmdHandler_SetTeam, 9},
     {{"setSuperGroup"}, &cmdHandler_SetSuperGroup, 9},
+    {{"settingsDump"}, &cmdHandler_SettingsDump, 9},
     {{"setu1"}, &cmdHandler_SetU1, 9},
     {{"setu2"}, &cmdHandler_SetU2, 9},
     {{"setu3"}, &cmdHandler_SetU3, 9},
@@ -430,6 +432,16 @@ void cmdHandler_SetSuperGroup(QString &cmd, Entity *e) {
     QString msg = "Set SuperGroup ID to: " + QString::number(val);
     qDebug() << msg;
     sendInfoMessage(MessageChannel::DEBUG_INFO, msg, src);
+}
+
+void cmdHandler_SettingsDump(QString &cmd, Entity *e) {
+    MapClient *src = e->m_client;
+
+    QString msg = "Sending settings config dump to console output.";
+    qDebug() << msg;
+    sendInfoMessage(MessageChannel::DEBUG_INFO, msg, src);
+
+    settingsDump(); // Send settings dump
 }
 
 // Slash commands for setting bit values

--- a/Projects/CoX/Servers/MapServer/SlashCommand.h
+++ b/Projects/CoX/Servers/MapServer/SlashCommand.h
@@ -47,6 +47,7 @@ void cmdHandler_FullUpdate(QString &cmd, Entity *e);
 void cmdHandler_HasControlId(QString &cmd, Entity *e);
 void cmdHandler_SetTeam(QString &cmd, Entity *e);
 void cmdHandler_SetSuperGroup(QString &cmd, Entity *e);
+void cmdHandler_SettingsDump(QString &cmd, Entity *e);
 void cmdHandler_SetU1(QString &cmd, Entity *e);
 void cmdHandler_SetU2(QString &cmd, Entity *e);
 void cmdHandler_SetU3(QString &cmd, Entity *e);

